packaging/debian/control

```diff
@@ -9,6 +9,7 @@ Build-Depends: debhelper-compat (= 12),
  libssl-dev,
  libtalloc-dev,
  pkgconf,
+ build-essential,
```

Comment:
Please don't do that. build-essential is implicitly present in all official Debian build environments, it shouldn't go into debian/control. You also don't have anything in your ci.yml workflow which would actually look at the declared build-deps here. Instead, add build-essential to the `sudo apt install` list in the workflow.